### PR TITLE
Remove legacy code and explicit pandas dependency

### LINKED ISSUE #230: prune dependencies
Let's see if we can depend on fewer things, e.g., remove sklearn and pandas

### PR DESCRIPTION
**Pull request by qubixes**
Fixes #230 

Removes wget, sklearn and pandas requirements.

### CODE REVIEW

File: metasyn/demo/dataset.py [hunk 1]
```diff
@@ -1,70 +1,69 @@
...
-    time_dist = DateTimeUniformDistribution.default_distribution()
-    dframe["Married since"] = [time_dist.draw() if np.random.rand() < 0.9 else pd.NA
-                               for _ in range(len(dframe))]
-
-    dframe["all_NA"] = [pd.NA for _ in range(len(dframe))]
-    # Remove some columns for brevity and write to a file.
-    dframe = dframe.drop(["SibSp", "Pclass", "Survived"], axis=1)
-    dframe.to_csv(output_fp, index=False)
-    return output_fp
+# import numpy as np
```
Comment (qubixes) [inline metasyn/demo/dataset.py:11]: I have kept this in, just in case.

**Review by vankesteren — APPROVED** [review]
The best type of PR. Love this!

**Comment by vankesteren** [comment]
🤩

**Comment by qubixes** [comment]
Yes, less code! Making up for my last PR!

**Pull request merged**